python/tank/bootstrap/manager.py

```diff
@@ -574,14 +593,109 @@ def _bootstrap_sgtk(self, engine_name, entity, progress_callback=None):
         else:
             raise TankBootstrapError("Unknown configuration update status!")
 
+        return config, status
+
+    def bootstrap_toolkit(self, entity):
```

Comment:
This is the reason: https://github.com/shotgunsoftware/tk-framework-desktopstartup/pull/33/files?diff=unified#diff-c55bf62c6517b7ddc8ab0c2633396f41R421